pkg/issuer/acme/http/http.go

```diff
@@ -42,8 +42,9 @@ const (
 	// acmeSolverListenPort is the port acmesolver should listen on
 	acmeSolverListenPort = 8089
 
-	domainLabelKey = "certmanager.k8s.io/acme-http-domain"
-	tokenLabelKey  = "certmanager.k8s.io/acme-http-token"
+	domainLabelKey               = "certmanager.k8s.io/acme-http-domain"
+	tokenLabelKey                = "certmanager.k8s.io/acme-http-token"
+	solverIdentificationLabelKey = "certmanager.k8s.io/solver-app"
```

Comment:
Can we rename this to `certmanager.k8s.io/acme-http01-solver: "true"`? I do like the idea of being able to encode the `solver type` into the annotations, but for the time being I think we should keep it simple 😄

Reply:
done